test/integration/discordactions.test.js

```diff
@@ -785,5 +785,24 @@ describe("Discord actions", function () {
       expect(res.body.message).to.be.equal("invite generated successfully");
       expect(res.body.inviteLink).to.be.equal("discord.gg/asdfdsfsd");
     });
+
+    it("should generate discord link if user is a superUser", async function () {
+      fetchStub.returns(
+        Promise.resolve({
+          status: 201,
+          json: () => Promise.resolve({ data: { code: "asdfdsfsd" } }),
+        })
+      );
+
+      mavenUserId = await addUser(mavenUser);
+      mavenAuthToken = authService.generateAuthToken({ userId: mavenUserId });
```

Comment:
what's the use of this part?

Reply:
There is no use, as there is a token already present reused that